src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int max_number = 100;
```

Comment:
Такие поля называются константами. Они один раз задаются при запуске программы и никогда не меняются. Для таких полей есть правило именования: имя должно быть из больших букв.
```suggestion
    private static final int MAX_NUMBER = 100;
```